pkg/config/config.go

```diff
@@ -229,6 +230,29 @@ func (c *HarvesterConfig) GetKubeletArgs() ([]string, error) {
 	return []string{}, nil
 }
 
+func (c *HarvesterConfig) Merge(other HarvesterConfig) error {
+	// We need to manually merge the "Networks" field (map[string]Network) beacuse Mergo won't
```

Comment:
```suggestion
	// We need to manually merge the "Networks" field (map[string]Network) because Mergo won't
```
nit: My editor found this typo.

Reply:
Fixed, thanks!